main/inc/lib/myspace.lib.php

```diff
@@ -1154,139 +1154,260 @@ public static function displayResumeLP(
+                        || !empty($sessionStudent)
+                    ) {
+                        $htmlData .= "<a href='#!' id='$hiddenFieldLink' onclick='showHideStudent(\"$hiddenField\")'>
+                        <div class='icon_add'>$iconAdd</div>
+                        <div class='icon_remove hidden'>$iconRemove</div>
+                        </a>
+                        <div id='$hiddenField' class='hidden'>";
+                        $studentRegistered = [];
+                        foreach ($courseStudent as $student) {
+                            $reportLink = Display::url(
+                                Display::return_icon('statistics.png', get_lang('Stats')),
+                                api_get_path(WEB_CODE_PATH).'mySpace/myStudents.php?details=true&student='.
+                                $student['id']
+                                ."&id_session=$lpSessionId"
+                                ."&course=$lpCourseCode"
+                            );
+                            if (!isset($studentRegistered[$student['id']])) {
+                                $studentRegistered[$student['id']][0] = $student;
+                                $htmlData .= "$reportLink ".$student['complete_name'].'<br>';
+                                $studentsName .= $student['complete_name'].' / ';
+                                $totalStudents++;
+                            }
+                        }
+                        foreach ($sessionStudent as $student) {
+                            $lpSessionId = (int) $student['session_id'];
+                            $reportLink = Display::url(
+                                Display::return_icon('statistics.png', get_lang('Stats')),
+                                api_get_path(WEB_CODE_PATH).'mySpace/myStudents.php?details=true&student='.
+                                $student['id']
+                                ."&id_session=$lpSessionId"
+                                ."&course=$lpCourseCode"
+                            );
+
+                            if (!isset($studentRegistered[$student['id']][$lpSessionId])) {
+                                $studentRegistered[$student['id']][$lpSessionId] = $student;
+                                $studentsName .= $student['complete_name'].' / ';
+                                $htmlData .= "$reportLink <strong>".$student['complete_name'].'</strong><br>';
+                                $totalStudents++;
+                            }
+                        }
+                        foreach ($sessionStudent as $student) {
+                            $reportLink = Display::url(
+                                Display::return_icon('statistics.png', get_lang('Stats')),
+                                api_get_path(WEB_CODE_PATH).'mySpace/myStudents.php?details=true&student='.
+                                $student['id']
+                                ."&id_session=".($lpSessionId)
```

Comment:
Verificar el valor correcto de $lpSessionId es 0?  Viene $lpSessionId = (int) $student['session_id']; ? 
Pero ese valor está dentro del for .... 

Revisar el código detenidamente, tener más cuidado en le definición de variables. 
Ya son más de 5 veces que estoy revisando este PR.

Reply:
No siempre es 0, cuando se suscribe un usuario a un lp lo puede hacer por un curso o por una sesion, En donde lo comentas, es necesario ya que viene dado por getCompanyLearnpathSubscription y alli se toma sesiones. La sesion tambien aplica para las clases.
 ** mismo contenido del comentario anterior **

Reply:
Acabas de corregir lo que indico en mi comentario en este commit ...

https://github.com/chamilo/chamilo-lms/pull/3773/files/720f59b9fdcfa9dcd8d1bfddb4e9e1a8ae875034..77df23a5940dad58b1f2e20a9232f78421ca956f

Por favor Carlos no discutas cada comentario que hago excusándote por x razones cuando indico que hay algo mal, corregirlo.